### PR TITLE
Add giflib

### PR DESCRIPTION
**Pull request by eustas**

### CODE REVIEW

**Review by fmeum — APPROVED** [review]

File: modules/giflib/5.2.1/presubmit.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+matrix:
+  platform:
+  - centos7
```
Comment (fmeum) [inline modules/giflib/5.2.1/presubmit.yml:3]:
```suggestion
```

Comment (eustas) [reply modules/giflib/5.2.1/presubmit.yml]: Done.

**Comment by fmeum** [comment]
`unistd.h` isn't available on Windows, is this missing some compatibility defines?

**Comment by eustas** [comment]
Honestly speaking, did not try building on windows. Let me figure out how to fix this.

**Comment by eustas** [comment]
Seems to work now. PTAL

**Pull request merged**